modules/t/test-genome-DBs/ontology/ontology/table.sql

```diff
@@ -197,6 +198,7 @@ CREATE TABLE `term` (
   `definition` text,
   `is_root` int(11) DEFAULT NULL,
   `is_obsolete` int(11) DEFAULT NULL,
+  `iri` varchar(1000) DEFAULT NULL,
```

Comment:
Likewise, would it be better to create iri as a TEXT instead of a VARCHAR if it is not going to be indexed?

Reply:
For term, this is more important, I'll do the modification today.

Reply:
Done